db/music.dbml

```diff
@@ -0,0 +1,299 @@
+Table submission {
+  submission_id uuid [pk, default: `gen_random_uuid()`]
+  participant_id int [not null, ref: > participant.participant_id]
+  bracket_id int [not null, ref: > bracket.bracket_id]
+  song_id int [not null, ref: > song.song_id]
+  order int [not null]
+  date_submitted timestamp [not null, default:`now()`]
+  active bool_int [not null]
+  is_dupe bool_int [not null]
+  title text [not null]
+  version text [null]
+  artist text [null]
+  collaborators text [null]
+  composer text [null]
+  link text [not null]
+  genre text [null]
+  emoji text [null]
+  comments text [null]
+
+  indexes {
+    (participant_id, bracket_id, song_id) [unique]
```

Comment:
I'd skip a unique index here and have separate indexes for `participant_id` and `bracket_id` (and maybe `song_id` too, but I dunno) since we'll likely want to be able to efficiently query by bracket and by submitter separately.

Reply:
You don't think there's value in enforcing uniqueness? I mean, it'll probably never happen, I guess.

Reply:
I'm not sure what the uniqueness here would really protect us against... the same person submitting the same song twice? I suppose it's possible, but I'm not sure it has to be caught at the database level, especially since there will need to be other checks anyway (like two songs per artist)

Reply:
Fair point.